sycl/include/CL/sycl/INTEL/fpga_lsu.hpp

```diff
@@ -73,21 +73,21 @@ template <class... mem_access_params> class lsu final {
 
 private:
   static constexpr int32_t _burst_coalesce_val =
-      GetValue<burst_coalesce_impl, mem_access_params...>::value;
+      _GetValue<burst_coalesce_impl, _mem_access_params...>::value;
```

Comment:
You also need to make changes in `fpga_extensions.hpp` to use the new names.
@MrSidims  are these changes allowable here at all?

Reply:
I don't mind renaming.

`@MrSidims are these changes allowable here at all?`
If you are worried about backwards compatibility: programs on FPGA are being AOT compiled, hence I'm not expecting any mismatches during linking between new and already existing SPIR-V modules (because for FPGA we don't have any stored).

`You also need to make changes in fpga_extensions.hpp to use the new names.`
I don't get it.

Reply:
> You also need to make changes in `fpga_extensions.hpp` to use the new names.
Fpga_extenstions.hpp doesn't have any references to the specific local variables or template params, it only includes other header files. Since I haven't changed any header file names, it shouldn't impact fpga_extensions.hpp

Reply:
>> You also need to make changes in fpga_extensions.hpp to use the new names.
> I don't get it.

Tests for one of the previous commits are failed due to use of `GetValue` instead of `_GetValue` in `fpga_extensions.hpp`.
These tests are: `fpga_io_pipes.cpp`, `fpga_pipes.cpp` and `fpga_lsu.cpp`.

Reply:
> > > You also need to make changes in fpga_extensions.hpp to use the new names.
> > > I don't get it.
> 
> Tests for one of the previous commits are failed due to use of `GetValue` instead of `_GetValue` in `fpga_extensions.hpp`.
> These tests are: `fpga_io_pipes.cpp`, `fpga_pipes.cpp` and `fpga_lsu.cpp`.

That should be addressed by this [commit](https://github.com/intel/llvm/pull/2375/commits/fe56227f4ed809a571929d4642412d42d495366a)

Reply:
Ok, then.